lib/syncthing/auditservice.go

```diff
@@ -10,56 +10,50 @@ import (
 	"encoding/json"
 	"io"
 
+	"github.com/thejerf/suture"
+
 	"github.com/syncthing/syncthing/lib/events"
+	"github.com/syncthing/syncthing/lib/util"
 )
 
 // The auditService subscribes to events and writes these in JSON format, one
 // event per line, to the specified writer.
 type auditService struct {
-	w       io.Writer     // audit destination
-	stop    chan struct{} // signals time to stop
-	started chan struct{} // signals startup complete
+	suture.Service
+	w       io.Writer // audit destination
+	sub     *events.Subscription
 	stopped chan struct{} // signals stop complete
 }
```

Comment:
I don't think we need the stopped anymore?

Reply:
Or perhaps we can scrap it?

Reply:
We can. I just accepted the comment on `WaitForStop` that we need it for testing, but we don't actually (`Stop` now only returns once it actually stopped).